### PR TITLE
Issue-504: modified two existing links

### LINKED ISSUE #504: Orphan Pages
## Orphan pages

Orphan pages :coffin: found on https://open-horizon.github.io/

[View Results](https://github.com/open-horizon/open-horizon.github.io/actions/runs/5139713086?check_suite_focus=true)

### PR DESCRIPTION
**Pull request by joewxboy**
# Pull Request Template

## Description

Fixes #504

...

### CODE REVIEW

**Comment by joewxboy** [comment]
All, I just need a Maintainer to review since I don't want to merge this without another set of eyes on it.

**Comment by johnwalicki** [comment]
Hi @joewxboy  - I looked at this patch over the weekend.  Didn't approve.  I don't think your PR will actually fix the orphan page scan because it might not scan the _config.yml file.   We can try again, but my confidence is low.

**Review by johnwalicki — APPROVED** [review]
I looked at this patch over the weekend.  Didn't approve.  I don't think your PR will actually fix the orphan page scan because it might not scan the _config.yml file.   We can try again, but my confidence is low.

**Pull request merged**
